src/Carbon/Carbon.php

```diff
@@ -396,11 +396,7 @@ public function __construct($time = null, $tz = null)
             )
         ) {
             $dateTime = new DateTime('now', $timezone);
-            $microTime = microtime(true) * 1000000 % 1000000;
-            if ($microTime > 0) {
-                $microTime = str_pad(strval($microTime), 6, '0', STR_PAD_LEFT);
-                $time = $dateTime->format(static::DEFAULT_TO_STRING_FORMAT).'.'.$microTime;
-            }
+            $time = $dateTime->format(static::DEFAULT_TO_STRING_FORMAT).substr(microtime(), 1, 6);
```

Comment:
If you will use dot from a microtime(), then you should cut string 7 char long: 
```php
.......FORMAT).substr(microtime(), 1, 7);
```

Reply:
Thanks^^ Strange that unit tests passed.

Reply:
`.12345` or `.123456` is both valid :)